[12.0][REF] include representative address type in cooperator

victor-champonnois opened the pull request:
Fix https://github.com/coopiteasy/vertical-cooperative/issues/350
related to https://github.com/coopiteasy/addons/pull/261
14.0 version: #363

robinkeunen reviewed (approved):
LGTM 👍 Just check that deploying https://github.com/coopiteasy/addons/pull/261 at the same time does not delete data in column `type`. I'm 90% sure it does not but worth a check.

victor-champonnois commented:
> LGTM +1 Just check that deploying [coopiteasy/addons#261](https://github.com/coopiteasy/addons/pull/261) at the same time does not delete data in column `type`. I'm 90% sure it does not but worth a check.

I looked at the demain DB. I compared the prod with the test. There is exactly the same number of type "representative" and type "contact" in both DB, so there is no data loss (at least for this DB)

victor-champonnois committed:
[REF] include representative address type in cooperator

Put the representative type in cooperator module instead of in
partner_contact_address for inclusion in OCA.

huguesdk commented:
/ocabot merge minor
